src/content/docs/en/pages/guides/marketplace/templates/nuxt-notes.mdx

```diff
@@ -0,0 +1,93 @@
+
+This template uses Nuxt version `3.0.0`.
+
+:::note
+This is a community template. It was created by third-party or external developers and Azion is only enabling the deployment in the Azion Edge Platform. Azion doesn't provide maintenance or support for the content itself but can assist you with any queries or issues you may encounter while deploying this template or using the platform.
+:::
+
+## Requirements
+
+- A [GitHub account](https://github.com/signup) to connect with Azion and create your new repository.
+    - Every push will be deployed automatically to this repository to keep your project updated.
+- This template uses [Application Accelerator](/en/documentation/products/build/edge-application/application-accelerator/), [Edge Functions](/en/documentation/products/build/edge-application/edge-functions/), and [Edge Cache](https://www.azion.com/en/documentation/products/build/edge-application/edge-cache/), and it could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information.
+
+---
+
+## Deploying the template
+
+You can obtain and configure your template through the Azion Console. To easily deploy it at the edge, click the button below.
+
+<LinkButton
+    label="Deploy"
+    link="https://console.azion.com/create/azion-community/nuxt-notes"
+    icon="ai ai-azion"
+    icon-pos="left"
+  />
+
+---
+
+## Setting up the template
+
+In the configuration form, you must provide the information to configure your application. Fill in the presented fields. 
+
+Fields identified with an asterisk are mandatory.
+
+1. Connect Azion with your GitHub account.
+    - A pop-up window will open to confirm the installation of the [Azion GitHub App](/en/documentation/products/guides/azion-github-app/), a tool that connects your GitHub account with Azion's platform.
+    - Define your permissions and repository access as desired.
+2. Select the **Git Scope** to work with.
+3. Define a name for your edge application.
+    - The bucket for storage and the edge function will use the same name.
+    - Use a unique and easy-to-remember name. If the name has already been used, the platform returns an error message.
+4. Click the **Deploy** button to start the deployment process.
+
+During the deployment, you'll be able to follow the process through a window showing off the logs. When it's complete, the page shows information about the application and some options to continue your journey.
+
+:::note
+The link to the edge application allows you to see it on the browser. However, it takes a certain time to propagate and configure the application in Azion's edge locations. It may be necessary to wait a few minutes for the URL to be activated and for the application page to be effectively displayed in the browser.
```

Comment:
```suggestion
The link to the edge application allows you to see it on the browser. However, it may take some time to propagate and configure the application in Azion's edge locations. It may be necessary to wait a few minutes for the URL to be activated and for the application page to be effectively displayed in the browser.
```